<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>choice</class>
 <widget class="QWidget" name="choice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>379</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QTreeView" name="appTree"/>
       </item>
       <!-- Disable until there is cooperation with kdevplatform/plugins/appwizard/kdevappwizard.knsrc
       <item>
        <widget class="QPushButton" name="getNewButton">
         <property name="text">
          <string>Get More Templates</string>
         </property>
         <property name="icon">
          <iconset theme="get-hot-new-stuff">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       -->
       <item>
        <widget class="QPushButton" name="installButton">
         <property name="text">
          <string>Install Template From File</string>
         </property>
         <property name="icon">
          <iconset theme="application-x-archive">
           <normaloff>.</normaloff>.</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="descriptionFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="pictureLabel">
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="fixedSize">
           <size>
            <width>256</width>
            <height>256</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="descriptionLabel">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Project name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="kcfg_appName"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
